### ISSUE TITLE
Autoloader.php references class files that aren't included in the repo. Causing fatal error

### ISSUE THREAD

**Issue by ayyobro** [dompdf/dompdf#1127]
Creating a website for someone who wants some html converted to PDF format

I keep getting these errors when trying to use it.

Warning: require_once(/htdocs/Roscoe/wp-content/themes/roscoe/includes/dom-pdf/lib/php-font-lib/src/FontLib/Autoloader.php): failed to open stream: No such file or directory in /htdocs/Roscoe/wp-content/themes/roscoe/includes/dom-pdf/autoload.inc.php on line 20

Fatal error: require_once(): Failed opening required 'htdocs/Roscoe/wp-content/themes/roscoe/includes/dom-pdf/lib/php-font-lib/src/FontLib/Autoloader.php' (include_path='.:/bin/php/php5.6.10/lib/php') in htdocs/Roscoe/wp-content/themes/roscoe/includes/dom-pdf/autoload.inc.php on line 20

I checked the source code that I pulled from github and confirmed that these were missing from the download. I checked the repo origin in github ( here in this repository ) and confirmed they were also missing from the source. 

When I removed the references to those files ( thinking maybe I don't need them ) I received an error that the Dompdf() class did not exist either. So this means the autoload failed. 

Am I missing a step here? 

Where are these files? Why aren't they in this repository? Why does no one mention this nuance in the documentation? 

Please help.

**Comment by bsweeney** [comment]
Where are you downloading dompdf from? The Autoloader.php file does not reference any non-existent libraries, but the autoload.inc.php file does. However, you should only be using that file if you're downloading the packaged release. If you're using composer you should use the composer autoloader and ignore the ones we include in the package. And if you're grabbing the dompdf source instead of the packaged release you'll have to manually download any of the dependencies (such as php-font-lib) manually.

**Comment by ayyobro** [comment]
Thanks for that. I pulled in the dependencies via composer and got those warnings out of the way, but I'm still having issues with it not finding the Dompdf() class. 

I even referenced the class file directly in my functions.php file ( I should have mentioned I'm using Wordpress ), but it still says the class doesn't exist. I confirmed that the path was accurate to the class file. 

Thoughts on that?
